content/en/docs/components/central-dash/github.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "GitHub Repository"
+description = "GitHub repository for Kubeflow Central Dashboard"
+weight = 999
+manualLink = "https://github.com/kubeflow/kubeflow/tree/master/components/centraldashboard"
+icon = "fa-brands fa-github"
++++
+
+The Kubeflow Central Dashboard is developed in the [`kubeflow/kubeflow`](https://github.com/kubeflow/kubeflow/tree/master/components/centraldashboard) repository.
```

Comment:
Do these pages work for you ?
When I click to the GitHub repository in the left menu it redirects me to the GitHub.
Or this page is for users who directly access it via URL ?
E.g.
https://deploy-preview-3640--competent-brattain-de2d6d.netlify.app/docs/components/central-dash/github/

Reply:
You are correct, these links are here not because they are accessible, but because Docsy (our template engine) requires them.

Users are actually redirected automatically when they click the sidebar link.